src/sonic-config-engine/minigraph.py

```diff
@@ -38,7 +38,7 @@
 FRONTEND_ASIC_SUB_ROLE = 'FrontEnd'
 BACKEND_ASIC_SUB_ROLE = 'BackEnd'
 BACKEND_ASIC_INTERFACE_NAME_PREFIX = 'Ethernet-BP'
-
+INTERNAL_BGP_SESSION_NAME_PREFIX = 'ASIC'
```

Comment:
can you remove since it is not used anymore.

Reply:
Removed in latest commit